src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -879,14 +594,11 @@ private void updateSubscribers()
             }
         }
 
-        isFieldsDirty = false;
         isDirty = false;
-        isCitizensDirty = false;
-        isBuildingsDirty = false;
         permissions.clearDirty();
 
-        buildings.values().forEach(AbstractBuilding::clearDirty);
-        citizens.values().forEach(CitizenData::clearDirty);
+        buildingManager.clearDirty();
```

Comment:
Is this method called from somewhere else? If not move it into sendPackets

Reply:
No, but it should also be called if no message has to be send in this case here.